backend/development.ini.sample

```diff
@@ -364,15 +364,10 @@ api.key = %(basic_setup.api_key)s
 
 ## User custom properties: This features allows defining custom instance-global fields on users of this instance.
 # See the Tracim settings documentation (available in source code, see "user_custom_properties.md") for more
-# information about it. By default, Tracim doesn't use custom properties.
-; user.custom_properties.json_schema_file_path =
-; user.custom_properties.ui_schema_file_path =
-; user.custom_properties.translations_dir_path =
-## example with user.custom_properties.dir shortcut:
-# user.custom_properties.dir = %(here)s/examples/user_custom_properties/default
-# user.custom_properties.json_schema_file_path = %(user.custom_properties.dir)s/schema.json
-# user.custom_properties.ui_schema_file_path = %(user.custom_properties.dir)s/ui.json
-# user.custom_properties.translations_dir_path = %(user.custom_properties.dir)s/locale
+# information about it.
+; user.custom_properties.json_schema_file_path = %(here)s/tracim_backend/templates/user_custom_properties/default/schema.json
+; user.custom_properties.ui_schema_file_path = %(here)s/tracim_backend/templates/user_custom_properties/default/ui.json
+; user.custom_properties.translations_dir_path = %(here)s/tracim_backend/templates/user_custom_properties/default/locale
```

Comment:
We need docker modification in common.sh for this kind of path (`%(here)s` not work in docker context) and also information in changelog about what we need to do when upgrading instance (when development.ini already exist.

I can done docker modification if you want.

Reply:
Done for docker.